src/dotnet/Program.cs

```diff
@@ -140,6 +140,8 @@ private static int ProcessArgs(string[] args)
             {
                 // mimic the env variable
                 Environment.SetEnvironmentVariable(CommandContext.Variables.Verbose, verbose.ToString());
+                // Use the current value of AnsiPassThru but make sure child processes have it set to true.
+                CommandContext.ShouldPassAnsiCodesThrough();
```

Comment:
I get the intent here but I don't get the side effect you're going after. 

You're forcing the variable to get read, but what does that affect?